src/implementation/Server/HTTPServer/HTTPServer.ts

```diff
@@ -124,7 +125,10 @@ export default class HTTPServer implements IServer {
   }
 
   async stop(): Promise<void> {
-    await this.server.close();
+    const s = this.server.getServer();
+    const httpTerminator = createHttpTerminator({ server: this.server.getServer() });
```

Comment:
We can cleanup `s` or use that in params. Also we can remove the comment.

Reply:
Good catch!